packages/scandipwa/src/component/ProductCustomizableOption/ProductCustomizableOption.component.js

```diff
@@ -253,10 +253,12 @@ export class ProductCustomizableOption extends PureComponent {
             renderOptionLabel,
             option: {
                 title,
-                data: {
-                    price_type,
-                    price
-                }
+                data: [
+                    {
+                        price_type = 'FIXED',
+                        price = 0
+                    } = {}
+                ] = []
```

Comment:
This is surprising! How did this change from an object to an array?

Reply:
This is the matter of unifying the data we are working with